src/apps/console/components/multi-step-progress-wrapper.tsx

```diff
@@ -1,17 +1,37 @@
-import { Link } from '@remix-run/react';
+import { Link, useLocation } from '@remix-run/react';
 import { ReactNode } from 'react';
 import { Button } from '~/components/atoms/button';
 import { ArrowLeft } from '~/console/components/icons';
+import { SignOut } from '@jengaicons/react';
+import { authBaseUrl } from '~/root/lib/configs/base-url.cjs';
+import { getCookie } from '~/root/lib/app-setup/cookies';
+import { useExternalRedirect } from '~/root/lib/client/helpers/use-redirect';
 import SplitWrapper from './split-wrapper';
 
 type ITitleSection = {
   title: ReactNode;
   subTitle: ReactNode;
 };
 const TitleSection = ({ title, subTitle }: ITitleSection) => {
+  const cookie = getCookie();
+  const { pathname } = useLocation();
+  const eNavigate = useExternalRedirect();
+
   return (
     <div className="flex flex-col gap-xl">
-      <div className="heading4xl text-text-default">{title}</div>
+      <div className="flex flex-row justify-between">
+        <div className="heading4xl text-text-default">{title}</div>
+        <Button
+          variant="plain"
+          suffix={<SignOut />}
+          size="sm"
+          content="Sign Out"
+          onClick={() => {
+            cookie.set('url_history', pathname);
```

Comment:
adding url_history from this page is not required.

Reply:
done